### PR TITLE
added some comments to classes and methods

### PR DESCRIPTION
**Pull request by rassl95**

### CODE REVIEW

File: src/main/java/com/github/hey_world_team/currency_converter/service/FileService.java [hunk 1]
```diff
@@ -75,19 +89,30 @@ private void prepareCollectionByDownloadFile() {
         }
     }
 
+    /**
+     * This method parses an XML file from the specified path
+     *
+     * @param path
```
Comment (dmitr2ish) [inline src/main/java/com/github/hey_world_team/currency_converter/service/FileService.java:95]: not completed comment

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -2,10 +2,15 @@
 server.servlet.context-path=/
 server.port=8081
 # postgres
-spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
+spring.datasource.url=jdbc:postgresql://localhost:5433/postgres
```
Comment (dmitr2ish) [inline src/main/resources/application.properties:5]: It's your local property, you do have not to push it to the general branch

File: src/main/resources/application.properties [hunk 3]
```diff
@@ -2,10 +2,15 @@
...
 spring.datasource.driver-class-name=org.postgresql.Driver
 spring.datasource.username=postgres
 spring.datasource.password=pass
+## history
+#spring.datasource2.url=jdbc:postgresql://localhost:5434/history
+#spring.datasource2.driver-class-name=org.postgresql.Driver
+#spring.datasource2.username=postgres
+#spring.datasource2.password=pass
```
Comment (dmitr2ish) [inline src/main/resources/application.properties:13]: pls delete it, also you dont need to create second db

File: src/main/resources/sql/schema2.sql [hunk 4]
```diff
@@ -0,0 +1,8 @@
+--CREATE TABLE IF NOT EXISTS history (
+--    id SERIAL PRIMARY KEY,
+--    conversion_date DATE NOT NULL,
+--    input_currency VARCHAR(3) NOT NULL,
+--    input_amount DECIMAL(18, 2) NOT NULL,
+--    output_currency VARCHAR(3) NOT NULL,
+--    output_amount DECIMAL(18, 2) NOT NULL
+--);
```
Comment (dmitr2ish) [inline src/main/resources/sql/schema2.sql:8]: pls delete it

**Review by dmitr2ish — CHANGES_REQUESTED** [review]
Need changes:
- do have not need to push your properties and commented properties;
- a lot of not completed java doc comments, try to describe what to do this method or class, also don't forget describe  arguments, exceptions and a return value;
- wrong naming of history service and history repository and name of saving history method;
- code style issues;
- For all new logic need to add unit tests:
 - history repository
 - history service

File: src/main/java/com/github/hey_world_team/currency_converter/repository/CurrencyRepositoryImpl.java [hunk 5]
```diff
@@ -98,10 +126,17 @@ public Currency getCurrencyById(String id) {
             return jdbcTemplate.queryForObject(selectQuery, new CurrencyMapper(), id);
         } catch (EmptyResultDataAccessException ex) {
             log.info("Currency with id {} not found", id);
-          return null;
+            return null;
```
[the remark this answers is not in the document]
Comment (rassl95) [reply src/main/java/com/github/hey_world_team/currency_converter/repository/CurrencyRepositoryImpl.java:129]: IDEA has fixed it like this

**Review by dmitr2ish — CHANGES_REQUESTED** [review]
Well done, you need to fix a little bit of the code style issues and update your idea code style of the project. Also, you need to be more careful with adding dependencies and check all changes that you want to push. And for future PRs, let's create different PR for different tasks.

**Review by dmitr2ish — APPROVED** [review]

**Pull request merged**